src/server.js

```diff
@@ -0,0 +1,10 @@
+const app = require('./app.js');
+require('env2')('config.env');
+
+app.set('PORT', process.env.PORT);
+app.listen(app.get('PORT'), () => {
+  // eslint-disable-next-line no-console
+  console.log(`The Server Is Up And Running On http://localhost:${app.get('PORT')}`);
+  // eslint-disable-next-line global-require
+  require('./mongoose.config');
```

Comment:
When requiring other javascript files, I would always add the `.js` extension, as you do for the `app` above. You can drop it, but if so, you should be dropping it everywhere (consistency is key)